File: basic/speed-loop.c
#include <stdio.h>
#include <time.h>

int main(){
  int cycles=1000000000; // 1 billion
  clock_t start, end;
  double duration;
  start=clock();
  for(int i=0; i<cycles; i++){
    //
  }
  end=clock();
  duration=(double)(end - start) / CLOCKS_PER_SEC;
  printf("C looped %d times in %lf secs\n", cycles, duration);
  return 0;
}

File: basic/speed-loop.js
// see how fast is node
function speed_loop(){
  const cycle=10000000000
  const start=Date.now()
  for(let i; i<cycle; i++){
    //
  }
  const end=Date.now()
  const duration=(end - start) / 1000
  console.log(`node looped %d times in %d secs`, cycle, duration)
}

speed_loop()